src/components/SUE/report/SueReportSend.tsx

```diff
@@ -26,44 +38,160 @@ export const SueReportSend = ({
           />
         </Wrapper>
-      )}
-    </>
+
+        {!isLoading && isDone && (
+          <Wrapper>
+            <Button
+              invert
+              notFullWidth
+              onPress={() => navigation.navigate(ScreenName.SueList)}
+              title={texts.sue.report.sendReportDone.toEntryList}
+            />
+
+            <Divider />
+
+            <View style={styles.feedbackContainer}>
+              <View style={styles.marginBottom24}>
+                <BoldText>{texts.sue.report.sendReportDone.feedbackHeader}</BoldText>
+              </View>
+
+              <View style={styles.marginBottom8}>
+                <RegularText small style={styles.marginBottom8}>
+                  {texts.sue.report.sendReportDone.ratingTitle}
+                </RegularText>
+
+                <Controller
+                  name="ratingCount"
+                  render={({ field: { onChange, value } }) => (
+                    <Rating
+                      imageSize={normalize(24)}
+                      onFinishRating={onChange}
+                      ratingColor={colors.primary}
+                      startingValue={value}
+                      style={styles.rating}
+                      tintColor={colors.lighterPrimary}
+                      type="custom"
+                    />
+                  )}
+                  control={control}
+                />
+              </View>
+
+              <View style={styles.marginBottom24}>
+                <Input
+                  control={control}
+                  inputContainerStyle={styles.inputContainer}
+                  inputStyle={styles.textArea}
+                  label={texts.sue.report.sendReportDone.messageTitle}
+                  multiline
+                  name="message"
+                  onFocus={() => scrollViewRef.current?.scrollTo({ x: 0, y: 250, animated: true })}
+                  placeholder={texts.sue.report.sendReportDone.messagePlaceholder}
+                  textAlignVertical="top"
+                />
+              </View>
+
+              <Button
+                disabled={loading}
+                onPress={handleSubmit(onSubmit)}
+                title={
+                  loading
+                    ? texts.feedbackScreen.sendButton.disabled
+                    : texts.sue.report.sendReportDone.sendButton
+                }
+              />
+            </View>
+          </Wrapper>
+        )}
+      </ScrollView>
+    </DefaultKeyboardAvoidingView>
   );
 };
 
 const styles = StyleSheet.create({
+  feedbackContainer: {
+    backgroundColor: colors.lighterPrimary,
+    borderRadius: normalize(8),
+    flex: 1,
+    marginTop: normalize(24),
+    padding: normalize(24)
+  },
   image: {
     alignSelf: 'center',
     height: normalize(50),
     width: '100%'
+  },
+  inputContainer: {
+    borderColor: colors.transparent
+  },
+  marginBottom8: {
+    marginBottom: normalize(8)
+  },
+  marginBottom24: {
+    marginBottom: normalize(24)
+  },
```

Comment:
could these be more like "containerXY" with margins instead of having random margin named styles? what if the value changes, the key needs to change as well, which would be heavy.